### PR TITLE
Lp/refactor/320/placeholders instead empty messages

### PR DESCRIPTION
**Pull request by HegelPro**

### CODE REVIEW

File: packages/jmix-front-generator/src/generators/react-typescript/app/template/i18n-message-packs/fr.json [hunk 1]
```diff
@@ -19,12 +19,12 @@
...
+    "file.download.notAllowed": "file.download.notAllowed [TRANSLATION NOT AVAILABLE]",
+    "file.noFile": "file.noFile [TRANSLATION NOT AVAILABLE]",
 
-    "list.doesNotExist": "[TRANSLATION NOT AVAILABLE]",
+    "list.doesNotExist": "list.doesNotExist [TRANSLATION NOT AVAILABLE]",
 
-    "editor.doesNotExist": "[TRANSLATION NOT AVAILABLE]",
+    "editor.doesNotExist": "editor.doesNotExist [TRANSLATION NOT AVAILABLE]",
 
     "cubaReact.dataTable.clearAllFilters": "Effacer les filtres",
```
Comment (web-devel) [inline packages/jmix-front-generator/src/generators/react-typescript/app/template/i18n-message-packs/fr.json:29]: let's also rename `cubaReact.` prefix to `jmix.`

File: example-react-app/src/app/AppMenu.tsx [hunk 2]
```diff
@@ -16,205 +16,205 @@ export const AppMenu = () => {
         screenId={"TestBlankComponent"}
         icon={<BarsOutlined />}
         caption={"menu.TestBlankComponent"}
-        key={"1214a4b6-7bf9-4f9b-ae4d-0593b09562a7"}
+        key={"63b4027b-99c1-4136-8367-01372551aac9"}
```
Comment (web-devel) [inline example-react-app/src/app/AppMenu.tsx:19]: Let's do not include this file in the changeset if there are no meaningful changes

**Review by web-devel — COMMENTED** [review]
Generally LGTM, see comments

**Review by vyacheslav-pushkin — COMMENTED** [review]
No further comments from my side.

**Pull request merged**